### PR TITLE
Update about page

### PR DESCRIPTION
**Pull request by oliverbarnes**
- Add team members (#7)
- Tweak mentions to liquidfeedback and plans for live api paid subscription

### CODE REVIEW

**Comment by jinjagit** [comment]
Looks good.

File: about.markdown [hunk 1]
```diff
@@ -4,10 +4,18 @@ title: About
 permalink: /about/
 ---
 
-A delegated voting service that aims to be easily plugged into proposal-making platforms of different kinds.
+**liquidvoting.io** is developed by a small team distributed across Barcelona, Paris, Morges (Switzerland) and Devon (UK):
```
Comment (niklaslong) [inline about.markdown:7]: I'd rather have Lausanne there (bigger, people are more likely to know it).
```suggestion
**liquidvoting.io** is developed by a small team distributed across Barcelona, Paris, Lausanne (Switzerland) and Devon (UK):
```

**Comment by niklaslong** [comment]
@oliverbarnes this is just hosted on gh-pages I presume? If so I'll merge 🚀

**Comment by oliverbarnes** [comment]
> @oliverbarnes this is just hosted on gh-pages I presume? If so I'll merge 🚀

yes, exactly!

**Pull request merged**